inst/parameters/TY28_BEFU24.yaml

```diff
@@ -56,36 +56,36 @@ FamilyAssistance_Abatement_AbatementScale:
   - 0.0
   - 0.27
 FamilyAssistance_Abatement_Order: 0.0
-FamilyAssistance_FTC_Rates_FirstChild: 7963.0
-FamilyAssistance_FTC_Rates_SubsequentChild: 6488.0
-FamilyAssistance_IWTC_Rates_UpTo3Children: 3770.0
+FamilyAssistance_FTC_Rates_FirstChild: 8056.0
+FamilyAssistance_FTC_Rates_SubsequentChild: 6563.0
+FamilyAssistance_IWTC_Rates_UpTo3Children: 5070.0
 FamilyAssistance_IWTC_Rates_SubsequentChildren: 780.0
 FamilyAssistance_IWTC_Eligibility: 1.0
 FamilyAssistance_FullTimeWorkingHours_Couple: 30.0
 FamilyAssistance_FullTimeWorkingHours_Single: 20.0
-FamilyAssistance_IWTC_IncomeThreshold_Single: 1184.94
-FamilyAssistance_IWTC_IncomeThreshold_Couple: 1184.94
+FamilyAssistance_IWTC_IncomeThreshold_Single: 1208.43
+FamilyAssistance_IWTC_IncomeThreshold_Couple: 1208.43
 FamilyAssistance_IWTC_ToBeneficiaries: 0.0
 FamilyAssistance_IWTC_PhaseIn_Single:
   thresholds: 0.0
   rates: 0.0
 FamilyAssistance_IWTC_PhaseIn_Couple:
   thresholds: 0.0
   rates: 0.0
-FamilyAssistance_MFTC_Rates_MinimumIncome: 37648.0
+FamilyAssistance_MFTC_Rates_MinimumIncome: 35048.0
 FamilyAssistance_BestStart_Abatement_AbatementScale:
   thresholds:
   - 0.0
   - 79000.0
   rates:
   - 0.0
   - 0.21
-FamilyAssistance_BestStart_Rates_Age0: 4062.0
-FamilyAssistance_BestStart_Rates_Age1or2: 4062.0
+FamilyAssistance_BestStart_Rates_Age0: 4109.0
+FamilyAssistance_BestStart_Rates_Age1or2: 4109.0
```

Comment:
Confirmed that automatic indexation of BestStart matches that in BEFU24 parameters.